Automattic/jetpack
# Docker: Profiling: Set xdebug.profiler_enable_trigger = On

- Removes some duplicated lines on php.ini
- Set xdebug.profiler_enable_trigger = On

**How to test**

`yarn docker:stop`
`yarn docker:build`
`yarn docker:up`

Turn on profiling COOKIE:
<img width="164" alt="screen shot 2018-03-21 at 16 20 20" src="https://user-images.githubusercontent.com/104869/37732524-068f0b92-2d24-11e8-962f-22287e5eb141.png">

Check that a a profiling output "cachegrind.out.*" gets created inside `/var/www/html` === `./docker/wordpress`
